addons/addons.xml

```diff
@@ -86,6 +86,13 @@
     <description>Customizes battle chat messages.</description>
     <bugtracker>https://github.com/Byrth/Lua-Byrth/issues</bugtracker>
     <support>https://discord.gg/b275nMv</support>
+  </addon>
+   <addon>
```

Comment:
A single extra space snuck in, somehow :)

Reply:
done